lms/templates/email/instructor_email_digest/subject.jinja2

```diff
@@ -1 +1,7 @@
-There has been some activity in Hypothesis
+{% if total_annotations > 1 and courses|length > 1 -%}
+Hypothesis: {{ total_annotations }} annotations in {{ courses|length }} courses yesterday
+{%- elif total_annotations > 1 -%}
+Hypothesis: {{ total_annotations }} annotations in 1 course yesterday
+{%- else -%}
+Hypothesis: 1 annotation yesterday
+{%- endif %}
```

Comment:
Note the `{%-` and `-%}`'s to make sure there's no leading or trailing newlines in the rendered subject